### PR TITLE
Fetch data files directly from repo in our tutorials

### PR DESCRIPTION
**Pull request by andped10**
To make the tutorials more transferable all needed input (experimental data) is now fetched using pooch. 

pooch has a local cache where it places the files when they are downloaded.  This ensures that it only fetches the files if have not been downloaded or changed (hash has changed).

### CODE REVIEW

File: docs/src/tutorials/sample/example.ort
Comment (andped10) [inline docs/src/tutorials/sample/example.ort:1]: This is a copy of the file in fitting that is no longer needed

**Review by henrikjacobsenfys — APPROVED** [review]
Looks good, the examples run on my computer without problems

File: docs/src/tutorials/sample/material_solvated.ipynb [hunk 1]
```diff
@@ -77,17 +80,22 @@
...
     "For information about the data being read in and the details of the model see [the previous tutorial](../fitting/simple_fitting.rst). \n",
     "We will gloss over these details here.\n",
-    "Use link to [download](example.ort) the ort data."
+    "We use `pooch` to fetch the file from the repository."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
```
Comment (rozyczko) [inline docs/src/tutorials/sample/material_solvated.ipynb:88]: This line and line 28 show that the notebook cells have been run before committing. Ideally, we want the notebook to have all outputs cleared.

**Review by rozyczko — APPROVED** [review]
Looks good.

File: docs/src/tutorials/sample/material_solvated.ipynb (hunk 1 above)
Comment (andped10) [reply docs/src/tutorials/sample/material_solvated.ipynb]: Thanks, should be clean now 🧼

**Pull request merged**